conf/application.conf

```diff
@@ -314,7 +314,11 @@ controllers.test.TestSetupController = {
 
 noneOnsSicCodes = "OTk5OTksNzQ5OTA="
 
-accessibility-statement.service-path = "/vat-registration"
+accessibility-statement {
+    host = "http://localhost:12346"
+    path = "/accessibility-statement"
```

Comment:
Looks like you don't use path & service path - can be removed

Reply:
Updated